stage0/log.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stage0
+
+import (
+	"os"
+
+	rktlog "github.com/coreos/rkt/pkg/log"
+)
+
+var log *rktlog.Logger
+
+func init() {
+	log = rktlog.New(os.Stderr, "rkt stage0", false)
```

Comment:
Maybe just "stage0" prefix? And maybe add a comment that debug mode is set in `InitDebug` function?